node-placeholder-scaler/scaler/scaler.py

```diff
@@ -74,7 +260,12 @@ def main():
 
     args = argparser.parse_args()
 
+    
+    namespace = "node-placeholder"
```

Comment:
we shouldn't hardcode the namespace that the node-placeholder will run in.  for example, i'm currently using the `cal-icor` fork of this chart in our hub deployment and it's running now in the support namespace.

see:
https://github.com/cal-icor/cal-icor-hubs/blob/staging/support/requirements.yaml#L2
https://github.com/cal-icor/cal-icor-hubs/blob/staging/support/values.yaml#L180

perhaps you can use the k8s label name?  `app.kubernetes.io/name=node-placeholder-scaler`